internal/queue/stub/stub.go

```
@@ -58,6 +58,9 @@ func (h hollowEnqueuer) Enqueue(_ context.Context, _ *worker.Job) error {
 }
 
 func (s *stubEnqueuer) Enqueue(ctx context.Context, job *worker.Job) error {
+	if job == nil {
+		return nil
```

Shouldn't we return an error here if the job is nil?